images/nginx/rootfs/build.sh

```diff
@@ -525,10 +539,13 @@ apt-get remove -y --purge \
   python \
   xz-utils \
   bc \
+  sensible-utils \
```

Comment:
Why are we introducing this?

Reply:
No, that is being removed :stuck_out_tongue: 
It's a transitive dependency not removed by the `autoremove`

Reply:
done